metrics/metrics.go

```diff
@@ -0,0 +1,66 @@
+package metrics
+
+import (
+	"encoding/json"
+	"io"
+	"strings"
+
+	metrics "github.com/rcrowley/go-metrics"
+)
+
+const (
+	GRPC = "grpc"
```

Comment:
__[golint]__
* [golint] exported const GRPC should have comment (or a comment on this block) or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->